veggieseasons/lib/data/local_veggie_provider.dart

```diff
@@ -75,7 +148,44 @@ class LocalVeggieProvider {
       vitaminCPercentage: 15,
       servingSize: '5 spears',
       caloriesPerServing: 20,
-      trivia: [],
+      trivia: [
+        Trivia(
+          'The nodules at the tip of an asparagus spear are actually what?',
```

Comment:
=> The nodules at the tip of an asparagus spear are actually which part of the plant?